OpenEdXMobile/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -75,7 +79,7 @@ private void init() {
 
         // initialize Fabric
         if (config.getFabricConfig().isEnabled() && !BuildConfig.DEBUG) {
-            Fabric.with(this, new CrashlyticsCore(), new Answers());
```

Comment:
Why are we removing Answers? I think CrashlyticsCore only includes crash reporting.